src/java/org/apache/cassandra/cql3/selection/ColumnFilterFactory.java

```diff
@@ -62,25 +63,27 @@ public static ColumnFilterFactory fromColumns(TableMetadata table,
      * @param table the table metadata
      * @param factories the {@code SelectorFactories}
      * @param orderingColumns the columns used for ordering
-     * @param nonPKRestrictedColumns the non primary key columns that have been resticted in the WHERE clause
+     * @param nonPKRestrictedColumns the non primary key columns that have been restricted in the WHERE clause
+     * @param returnStaticContentOnPartitionWithNoRows {@code true} if the query will return the static content when the partition has no rows, {@code false} otherwise.
```

Comment:
Nit: we can break this line
```suggestion
     * @param returnStaticContentOnPartitionWithNoRows {@code true} if the query will return the static content when the 
     * partition has no rows, {@code false} otherwise.
```